octobot/Cargo.toml

```diff
@@ -38,6 +38,7 @@ chrono = "0.4.19"
 prometheus = "0.13.1"
 maplit = "1.0.2"
 futures = "0.3.28"
+ttl_cache = "0.5.1"
```

Comment:
taking a quick look on github, it looks like this hasn't been touched for a few of years. 

i think this shouldn't be too hard to implement as a simple wrapper around a `hashmap` and `Instant` (for a monotonic clock)

Reply:
maybe something like this: https://github.com/tanium/octobot/blob/b1d10e680784111bff56de15b4e09d5e3f8e200d/lib/src/config.rs#L24C22-L25

where the cache object is internally threadsafe -- like, the `rwlock` or mutex is internal to the `Config` object.

```rust
struct TeamsCache {
    members: Mutex<std::collections::hashmap<String, TeamInfo>>,
}
```